rust/processor/src/models/stake_models/current_delegated_voter.rs

```diff
@@ -71,48 +71,53 @@ impl CurrentDelegatedVoter {
         txn_timestamp: chrono::NaiveDateTime,
         vote_delegation_handle_to_pool_address: &VoteDelegationTableHandleToPool,
         conn: &mut PgPoolConnection,
-    ) -> anyhow::Result<Option<Self>> {
+    ) -> anyhow::Result<CurrentDelegatedVoterMap> {
+        let mut delegated_voter_map: CurrentDelegatedVoterMap = HashMap::new();
+
         let table_item_data = write_table_item.data.as_ref().unwrap();
         let table_handle = standardize_address(&write_table_item.handle);
-        if let Some(VoteDelegationTableItem::VoteDelegationMap(inner)) =
+        if let Some(VoteDelegationTableItem::VoteDelegationMap(vote_delegation_maps)) =
```

Comment:
could we rename this to `VoteDelegationVector` or something since it's not a map